CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ release.
     - `type` to `process.context_switch_type`
   - Rename attributes for `process.paging.faults`
     - `type` to `process.paging.fault_type`
+- Move `process.cpu.state` to process registry
+  ([#681](https://github.com/open-telemetry/semantic-conventions/pull/681))
```

Comment:
Not sure if it should go to the user-facing changelog. I think we should ignore the changelog for this type of changes when no-op for the end users

Reply:
That makes sense. Removed in [b72b784](https://github.com/open-telemetry/semantic-conventions/pull/681/commits/b72b7845a20b197cfd104cb4ba4dad3b70394329)